llvm/test/CodeGen/AMDGPU/amdgpu-demote-scc-branches.ll

```diff
@@ -0,0 +1,256 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn -mcpu=gfx1010 < %s | FileCheck -check-prefixes=GFX10,GFX1010 %s
```

Comment:
Maybe add a gfx9 run line for good measure?

Reply:
Added gfx90a.